tests/smoke/.makim-scheduler.yaml

```
@@ -32,4 +32,4 @@ scheduler:
 
   test_sleep_function:
     schedule: "*/15 * * * *"
-    task: test.test-sleep
+    task: test.test-sleep
```

```suggestion
    task: test.test-sleep


```

remember to configure your IDE to keep an empty line at the end of the file